qi-test/tests/compiler/rules/deforest.rkt

```diff
@@ -9,7 +9,7 @@
          ;; necessary to correctly expand the right-threading form
          qi/flow/extended/forms
          qi/flow/core/compiler
-         qi/flow/core/deforest
+         qi/flow/core/passes/pass-0100-deforest
```

Comment:
These fixes (9d46ed0e824826fc033cbaae05f1cbc5e2296bba) belong with the commit that introduced the problem (probably 2247005
(Preliminary splitting of the compiler into separate modules for separate
passes., 2024-05-05)?)